Sources/Controllers/TargetMenu/PointEditing/OAFavoriteEditingHandler.mm

```diff
@@ -159,7 +159,6 @@ - (void)savePoint:(OAPointEditingData *)data newPoint:(BOOL)newPoint
     {
         [OAFavoritesHelper editFavoriteName:_favorite newName:data.name group:data.category descr:[_favorite getDescription] address:[_favorite getAddress]];
     }
-    [OAFavoritesHelper loadFavorites];
```

Comment:
This is not the fix for this issue. You need t find out why the calendar is called in the first place and add a check to call it only if we import/export parking points